app/screens/post_options/post_options.js

```diff
@@ -38,12 +38,12 @@ export default class PostOptions extends PureComponent {
         canFlag: PropTypes.bool,
         canPin: PropTypes.bool,
         canEdit: PropTypes.bool,
+        canMarkAsUnread: PropTypes.bool, //#backwards-compatibility:5.18v
```

Comment:
Is this something we're doing now? That's an interesting idea to stop forgetting about compatibility stuff

Reply:
Yup, i wanted to make it consistent. This way we can go back whenever we do an ESR update to remove these

Reply:
> Is this something we're doing now

@hmhealey Not until now. This is the first PR to make this. Let me know if you have suggestion on the format.
@enahum ☝️

Reply:
0/5 on the format 5/5 to have it